engine/src/main/java/org/camunda/bpm/engine/ExternalTaskService.java

```diff
@@ -398,6 +399,19 @@
    */
   public ExternalTaskQuery createExternalTaskQuery();
 
+  /**
+   * Returns a list of distinct topic names.
+   * Returns an empty list if no topics are found.
+   */
```

Comment:
The Javadoc should be more precise in which kind of topic names are returned (e.g. people might wonder if it returns the topics of any external task that is in a deployed process definition, or if it only returns the names of currently active. task instances)